ruby-challenges.rb

```diff
@@ -11,8 +11,22 @@
 # Expected output: '221 is odd'
 
 
+def even_or_odd number
+    if number % 2 == 0
```

Comment:
This looks great!  Try playing around with the many different Ruby methods available such as .even? or .odd?